psqlgraph/edge.py

```diff
@@ -100,6 +100,25 @@ def __init__(self, src_id=None, dst_id=None, properties={},
         else:
             self.dst_id = dst_id
 
+    def to_json(self):
+        return {
+            'src_id': self.src_id,
+            'dst_id': self.dst_id,
+            'label': self.label,
+            'acl': self.acl,
+            'properties': self.properties,
```

Comment:
Most of the time, these will be empty, would it be worth eliding zero values and doing a `edge_json.get('system_annotations', {})`?

Probably not, would take about 18k nodes to save 1MB this way...